bin/opencollective.js

```diff
@@ -0,0 +1,41 @@
+const child_process = require("child_process");
+const chalk = require("chalk");
+
+function padding(length) {
+	var padding = "";
+	for (var i = 0; i < length; i++) {
+		padding += " ";
+	}
+	return padding;
+}
+
+// Only show emoji on OSx (Windows shell doesn't like them that much ¯\_(ツ)_/¯ )
+function emoji(emoji) {
+	if (process.stdout.isTTY && process.platform === "darwin") {
+		return emoji;
+	} else {
+		return "";
+	}
+}
+
+function print(str, color) {
```

Comment:
```suggestion
function print(str = '', color = 'dim') {
```

It's supported by node 6 :)